### PR TITLE
Set max power on Aquaero

### PR DESCRIPTION
**Pull request by leoratte**
Set max power an Aquaero when changing PWM to allow fans to reach maximum speed.

### Checklist:

- [x] My code follows Linux code style guidelines
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested my changes on the affected device(s) - note on which
 Tested an Aquaero 5

### CODE REVIEW

File: aquacomputer_d5next.c [hunk 1]
```diff
@@ -1455,8 +1456,16 @@ static int aqc_write(struct device *dev, enum hwmon_sensor_types type, u32 attr,
 				ctrl_values[2] = 0;
 				ctrl_values_types[2] = AQC_BE16;
 
+				/* Set maximum power to 255 to allow the fan to
```
[the remark this answers is not in the document]
Comment (leoratte) [reply aquacomputer_d5next.c]: fixed

**Comment by aleksamagicka** [comment]
Tested on Aquaero 6 as well.

File: aquacomputer_d5next.c [hunk 2]
```diff
@@ -1455,8 +1456,17 @@ static int aqc_write(struct device *dev, enum hwmon_sensor_types type, u32 attr,
 				ctrl_values[2] = 0;
 				ctrl_values_types[2] = AQC_BE16;
 
+				/*
+				 * Set maximum power to 255 to allow the fan to
```
Comment (aleksamagicka) [inline aquacomputer_d5next.c:1460]: Sorry, just noticed - I think it's better to say `to 100%`?

**Comment by aleksamagicka** [comment]
Thanks!

**Pull request merged**